src/main/resources/application.properties

```diff
@@ -1,14 +1,15 @@
 spring.h2.console.enabled=true
 
-axa.jwt.secret=secret_key
-
-spring.cache.jcache.config=classpath:ehcache.xml
-
-#logging.level.org.springframework.web=INFO
-#logging.level.org.hibernate=ERROR
-#logging.level.net.guides=DEBUG
-
 server.port = 8082
 
-logging.file=jpn-lang-api.log
-logging.path=D:\1_Repo\jpn-lang-api
\ No newline at end of file
+logging.file=nihongo-api.log
+logging.path=D:\1_Repo\nihongo-api
+
+#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+#spring.datasource.url=jdbc:mysql:///nihongo_api
```

Comment:
you can remove all these commented lines

Reply:
removed